Gems/Atom/RHI/Code/Source/RHI/FrameGraphExecuteGroup.cpp

```diff
@@ -41,10 +42,13 @@ namespace AZ
             descriptor.m_scopeId = request.m_scopeId;
             descriptor.m_commandListCount = request.m_commandListCount;
 
+            uint32_t submitCount = request.m_submitCount;
+            uint32_t commandListCount = request.m_commandListCount;
             for (uint32_t i = 0; i < request.m_commandListCount; ++i)
             {
                 descriptor.m_commandListIndex = i;
                 descriptor.m_commandList = request.m_commandLists[i];
+                descriptor.m_submitRange = { (i * submitCount) / commandListCount, ((i + 1) * submitCount) / commandListCount };
```

Comment:
Can you add a comment explaining the submit range per command list. Basically explaining how each command list is assigned to do 'submitCount/commandListCount' worth of work.

Reply:
Added comment.